node/src/cli.rs

```diff
@@ -16,6 +39,66 @@ type RunCmd = cumulus_client_cli::RunCmd;
 #[cfg(not(feature = "parachain"))]
 type RunCmd = sc_cli::RunCmd;
 
+pub fn load_spec(
+    id: &str,
+    #[cfg(feature = "parachain")] parachain_id: cumulus_primitives_core::ParaId,
+) -> Result<Box<dyn sc_service::ChainSpec>, String> {
+    Ok(match id {
+        #[cfg(feature = "with-battery-station-runtime")]
+        "" | "dev" => Box::new(crate::chain_spec::dev_config(
+            #[cfg(feature = "parachain")]
+            parachain_id,
+        )?),
+        "battery_station" => Box::new(crate::chain_spec::DummyChainSpec::from_json_bytes(
+            #[cfg(feature = "parachain")]
+            &include_bytes!("../res/bs_parachain.json")[..],
+            #[cfg(not(feature = "parachain"))]
+            &include_bytes!("../res/bs.json")[..],
+        )?),
+        #[cfg(feature = "with-battery-station-runtime")]
+        "battery_station_staging" => Box::new(crate::chain_spec::battery_station_staging_config(
+            #[cfg(feature = "parachain")]
+            parachain_id,
+        )?),
+        #[cfg(not(feature = "with-battery-station-runtime"))]
+        "battery_station_staging" => panic!("{}", BATTERY_STATION_RUNTIME_NOT_AVAILABLE),
+        "zeitgeist" => Box::new(crate::chain_spec::DummyChainSpec::from_json_bytes(
+            #[cfg(feature = "parachain")]
+            &include_bytes!("../res/zeitgeist_parachain.json")[..],
+            #[cfg(not(feature = "parachain"))]
+            &include_bytes!("../res/zeitgeist.json")[..],
+        )?),
+        #[cfg(feature = "with-zeitgeist-runtime")]
+        "zeitgeist_staging" => Box::new(crate::chain_spec::zeitgeist_staging_config(
+            #[cfg(feature = "parachain")]
+            parachain_id,
+        )?),
+        #[cfg(not(feature = "with-zeitgeist-runtime"))]
+        "zeitgeist_staging" => panic!("{}", ZEITGEIST_RUNTIME_NOT_AVAILABLE),
+        path => {
+            let spec = Box::new(crate::chain_spec::DummyChainSpec::from_json_file(
+                std::path::PathBuf::from(path),
+            )?) as Box<dyn ChainSpec>;
+
+            match spec {
+                spec if spec.is_zeitgeist() => {
+                    #[cfg(feature = "with-zeitgeist-runtime")]
+                    // return Ok(Box::new(chain_spec::zeitgeist::ZeitgeistChainSpec::from_json_file(path)?));
```

Comment:
can this be removed?

Reply:
Mitigated in [6b5014c](https://github.com/zeitgeistpm/zeitgeist/pull/721/commits/6b5014c9bf89e28e432f24d01a08ae8e9e495b49)